ooniprobe/Utility/Tests.h

```diff
@@ -3,6 +3,8 @@
 
 @interface Tests : NSObject
 @property (strong, nonatomic) NSMutableArray *availableNetworkMeasurements;
+@property (strong, nonatomic) NSMutableDictionary *availableNetworkMeasurementsDic;
```

Comment:
It's generally more common to refer to "dictionaries" as "dict" rather than "dic".

Reply:
I also don't see where this dictionary is even being used. Searching in this PR I don't see it used anywhere.

Reply:
in fact is not. thanks for pointing it out